doc/modules/ROOT/pages/graph-analytics-serverless.adoc

```diff
@@ -1,10 +1,10 @@
-= GDS Sessions
+= Graph Analytics Serverless
+
+Aura Graph Analytics Serverless is an on-demand ephemeral compute environment for running GDS workloads. Each compute unit is called a GDS Session. It is offered as part of Neo4j Aura, a fast, scalable, always-on, fully automated cloud graph platform.
```

Comment:
❓ do we usually try to keep single sentence per line? git diffs looks better later on this way

Reply:
Yes we have an old convention of one sentence per line